### PR TITLE
Fix incorrect layout name assigned to main incident type

### PR DESCRIPTION
**Pull request by nericksen**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Fix bug where layout name was incorrect for main incident type

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by nericksen** [comment]
found a couple other bugs/improvements going through a clean install.  will push them today and confirm when fixes are all complete.

**Comment by nericksen** [comment]
Ok bug fixes have been implemented.

File: Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (tomer-pan) [inline Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md:1]: @nericksen I believe that you should raise the version here to 1.0.1 with the relevant changes

**Comment by tomer-pan** [comment]
@nericksen Great pack! We are looking into adding it to the phishing flow as well.
I added one comment regarding the version - you may want to reaise it to 1.0.1 after these changes.
I also think that it would be great if you will use demisto-sdk in order to create README files for the playbooks.

File: Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md (hunk 1 above)
Comment (eyalpalo) [reply Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md:1]: @nericksen @tomer-pan agreed. Please bump version of this pack and update the relevant release note with all the changed made.

**Comment by nericksen** [comment]
Thanks @tomer-pan !! I have updated the version as requested and generated the playbook docs :)

**Comment by tomer-pan** [comment]
> Thanks @tomer-pan !! I have updated the version as requested and generated the playbook docs :)

What I meant was that you should keep the 1.0.0 release notes and create a new 1.0.1 release notes file with the latest changes.
You can have a look at other packs release notes to see how it looks like.

**Comment by nericksen** [comment]
I didn't see any other packs with 1_0_0.md release notes so hopefully this is okay! Let me know

**Comment by tomer-pan** [comment]
@nericksen @eyalpalo all looks good to me now.

File: Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md (hunk 1 above)
Comment (eyalpalo) [reply Packs/SocialEngineeringDomainAnalysis/ReleaseNotes/1_0_0.md:1]: Hi @nericksen, the release note for version 1.0.0 wont be displayed anywhere as we don't use this file (this is the reason you cant find it in other packs).
You can either discard it or add relevant information from this file to the pack description.

**Review by eyalpalo — CHANGES_REQUESTED** [review]

**Review by eyalpalo — APPROVED** [review]

**Pull request merged**